doc/userguide/src/ExtendingRobotFramework/CreatingTestLibraries.rst

```diff
@@ -469,6 +489,17 @@ For example, the library below implements only keywords
 
 __ https://docs.python.org/tutorial/modules.html#importing-from-a-package
 
+Another way to avoid methods becoming keywords is to use `@library` class
+decorator which by default set its parameter `robot_auto_kws` to False.
+
+ .. sourcecode:: python
+
+     @library(robot_auto_kws=False)
+    class MyLibrary:
+
+         def my_keyword(self):
+            print('This method will not become keyword)
+
```

Comment:
This documentation can be fine tuned once PR #3219 adding `ROBOT_AUTO_KEYWORDS` is merged. That will be soon.